code/FulpstationCode/Fulp_traitor_items/traitor_chaplain.dm

```diff
@@ -1,285 +1,646 @@
+	A.do_attack_animation(D, ATTACK_EFFECT_KICK)
+	to_chat(A, "<span class='danger'>You repulse [D]!</span>")
+	D.visible_message("<span class='warning'>[A] extends out his hand to [D], a force repulsing them!</span>", \
+					"<span class='userdanger'>[A] extends his hand to you, sending you flying!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	playsound(get_turf(A), 'sound/magic/fireball.ogg', 100, TRUE)
+	var/atom/throw_target = get_edge_target_turf(D, A.dir)
+	D.throw_at(throw_target, 7, 14, A)
+	D.apply_damage(15, BURN)
+	D.Paralyze(20)
+	D.fire_stacks += 8
+	D.IgniteMob()
+	log_combat(A, D, "Repulsed (Justiciar's Hand)")
+	return
+
+/datum/martial_art/justiciar_hand/proc/Firetouch(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	to_chat(A, "<span class='danger'>You burn [D] with your hand!</span>")
+	D.visible_message("<span class='warning'>[A] burns [D] with his glowing hand!</span>", \
+					"<span class='userdanger'>[A] burns you with a glowing hand!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	playsound(get_turf(A), 'sound/magic/clockwork/ratvar_attack.ogg', 100, TRUE)
+	D.apply_damage(30, BURN)
+	D.fire_stacks += 3
+	D.IgniteMob()
+	log_combat(A, D, "Firey Touch (Justiciar's Hand)")
+	return
+
+/datum/martial_art/justiciar_hand/proc/Braintaze(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_PUNCH)
+	to_chat(A, "<span class='danger'>You invade [D]'s mind!</span>")
+	D.visible_message("<span class='warning'>[A] influences [D] with his hands!</span>", \
+					"<span class='userdanger'>[A] invades your mind!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	playsound(get_turf(A), 'sound/magic/clockwork/invoke_general.ogg', 100, TRUE)
+	D.apply_damage(10, BURN, BODY_ZONE_HEAD)
+	D.adjustOrganLoss(ORGAN_SLOT_BRAIN, 40, 150)
+	D.adjustStaminaLoss(80)
+	D.Paralyze(4 SECONDS)
+	D.stuttering += 15
+	D.Jitter(1.5 SECONDS)
+	D.fire_stacks += 4
+	D.IgniteMob()
+	log_combat(A, D, "Braintazed (Justiciar's Hand)")
+	return
+
+/datum/martial_art/justiciar_hand/proc/Timestop(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.do_attack_animation(D, ATTACK_EFFECT_KICK)
+	to_chat(A, "<span class='danger'>The Justiciar aids you for a brief second, stopping time while you kick [D]!</span>")
+	D.visible_message("<span class='warning'>[A] briefly stops time as he kicks [D]!</span>", \
+					"<span class='userdanger'>[A] kicks you in almost instantly!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", COMBAT_MESSAGE_RANGE, A)
+	playsound(get_turf(A), 'sound/magic/timeparadox2.ogg', 100, TRUE)
+	D.apply_damage(40, BURN)
```

Comment:
me looking for any effect (other than flavor text) that is even remotely related to stopping time in this move:
![image](https://user-images.githubusercontent.com/42606352/91681536-5370dc00-eb14-11ea-960f-581a5b7954d0.png)

Reply:
like, come on, man, this is just burn damage, a few fire stacks, and an ignition, why is it called time stop?

Reply:
also, this kind of seems like it's just "fiery touch, but slightly better and it requires an extra hit to execute"